curated_transformers/tokenization/__init__.py

```diff
@@ -1,4 +1,7 @@
 from .bbpe_encoder import build_byte_bpe_encoder_loader_v1
 from .hf_loader import build_hf_piece_encoder_loader_v1
 from .sentencepiece_encoder import build_sentencepiece_encoder_loader_v1
-from .wordpiece_encoder import build_wordpiece_encoder_loader_v1
+from .wordpiece_encoder import (
+    build_bert_wordpiece_encoder,
```

Comment:
Any reason why we're including this function and not any of the other (`build_xxxpiece_encoder` functions)?

Reply:
Oh, good question. We should probably start exporting them, it's weird to export the loaders but not the constructors. Maybe in a separate PR?

Reply:
Sounds good!